thundra/plugins/invocation/invocation_plugin.py

```diff
@@ -24,48 +27,103 @@ def before_invocation(self, data):
         context = data['context']
         memory_size = getattr(context, 'memory_limit_in_mb', None)
         self.start_time = time.time() * 1000
+
+        function_name = getattr(context, constants.CONTEXT_FUNCTION_NAME, None)
+
+
+        self.start_time = int(time.time() * 1000)
+        active_span = self.tracer.get_active_span()
+
         self.invocation_data = {
             'id': str(uuid.uuid4()),
-            'transactionId': data['transactionId'],
-            'applicationName': getattr(context, 'function_name', None),
+            'type': "Invocation",
+            'agentVersion': '',
+            'dataModelVersion': constants.DATA_FORMAT_VERSION,
             'applicationId': utils.get_application_id(context),
+            'applicationDomainName': active_span.domain_name or '' if active_span is not None else '',
+            'applicationClassName': active_span.class_name or '' if active_span is not None else '',
+            'applicationName': function_name,
             'applicationVersion': getattr(context, constants.CONTEXT_FUNCTION_VERSION, None),
-            'applicationProfile': utils.get_environment_variable(constants.THUNDRA_APPLICATION_PROFILE, default=''),
-            'applicationType': 'python',
-            'duration': None,
+            'applicationStage': '',
+            'applicationRuntime': 'python',
+            'applicationRuntimeVersion': str(sys.version_info[0]),
+            'applicationTags': {},
+
+            'traceId': '',
```

Comment:
`traceId` can be tried to get from `data`. If it is exist (put by trace plugin), it is set here, otherwise, it can left as empty.